frontend/public/components/debug-terminal.tsx

```diff
@@ -153,7 +154,11 @@ export const DebugTerminal: React.FC<DebugTerminalProps> = ({ podData, container
 
   if (generatedDebugPodName) {
     if (err) {
-      return <DebugTerminalError error={err} />;
+      return (
+        <DebugTerminalError
+          error={typeof err === 'object' ? t('public~The debug pod failed.') : err}
```

Comment:
@spadgett - I thought this provided the cleanest visuals, but I am open to suggestions.

Reply:
@zherman0 Doesn't this provide an error object with a detail message? We should be showing the actual message.

Reply:
Using the err,message field now.